src/ui/shared/requests/useNativeBalance.ts

```diff
@@ -29,10 +31,16 @@ function useNativeAddressPosition({
   );
 
   return useMemo(() => {
+    const nativePositions =
+      value?.positions?.filter(
+        (item) =>
+          item.chain === chain.toString() &&
+          item.type === 'asset' &&
+          !item.protocol
+      ) ?? [];
+    invariant(nativePositions.length <= 1, 'multiple native positions');
```

Comment:
Again, I don't understand why we want to crash here. Let's assume backend sends an extra position by mistake. Does this prevent us from signing a transaction?

Reply:
I see, you're right. Replaced invariant error with console.warn